src/components/Preferences/ThemeChooser.tsx

```diff
@@ -1,28 +1,50 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
 import { useRecoilState } from 'recoil';
-import { TileButton } from 'shared/components/TileButton/TileButton';
 import { Theme, themeState } from 'state/preferences/themeAtom';
+import { setTheme } from '@ui5/webcomponents-base/dist/config/Theme';
+import { TileButton } from 'shared/components/TileButton/TileButton';
 import { ThemePreview } from './ThemePreview/ThemePreview';
+import { useTranslation } from 'react-i18next';
 
-const AVAILABLE_THEMES: Theme[] = ['light_dark', 'light', 'dark', 'hcw', 'hcb'];
+const AVAILABLE_THEMES: Theme[] = [
+  'light_dark',
+  'sap_horizon',
+  'sap_horizon_dark',
+  'sap_horizon_hcw',
+  'sap_horizon_hcb',
+];
 
 export default function ThemeChooser() {
   const { t } = useTranslation();
-  const [theme, setTheme] = useRecoilState(themeState);
+  const [theme, setUsedTheme] = useRecoilState(themeState);
+
+  const getCurrentTheme = () =>
+    window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
change like in the previous file
`const isSystemThemeDark = window.matchMedia('(prefers-color-scheme: dark)').matches;`